### PR TITLE
Branch: 20_admin_edits

### PR DESCRIPTION
**Pull request by JoelLindow**
Admin can: Log in
Admin can not: Edit other users information
Admin can not: View standard user dashboard
Admin can: Do what admin want!

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -29,7 +30,8 @@ def update
     @user.update(user_params)
     if @user.save
       flash[:notice] = "Thine information has been updated!"
-      redirect_to dashboard_path
+      redirect_to dashboard_path if current_user.default?
```
Comment (mdevoe12) [inline app/controllers/users_controller.rb]: can we possibly break out these redirects to separate methods

Comment (JoelLindow) [inline app/controllers/users_controller.rb]: Matt... I appreciate the hot Latin flair that you bring to our team. However, I just don't see a need for that breakout. I had a private meeting with Katie and Ellen, behind your back, and we agree this was just fine before you suggested modifications.
Thanks... but no thanks.

*shrug*

Comment (katiekeel) [inline app/controllers/users_controller.rb]: @joel 👍

Comment (emcooper) [inline app/controllers/users_controller.rb]: yeah, thanks for the input but we already had a meeting about this 💯

Comment (katiekeel) [inline app/controllers/users_controller.rb]: @mdevoe12 👎

Comment (mdevoe12) [inline app/controllers/users_controller.rb]: I get it....:-(

Comment (JoelLindow) [inline app/controllers/users_controller.rb]: Do you?

Comment (mdevoe12) [inline app/controllers/users_controller.rb]: ...

Comment (JoelLindow) [inline app/controllers/users_controller.rb]: Thanks for the merge Matt. Upon 1 more private meeting we've decided you will not be penalized for the obstruction.

**Pull request merged**